R/target_market_share.R

```diff
@@ -158,14 +158,16 @@ target_market_share <- function(data,
       )
     ) %>%
     select(-.data$target_name, -.data$green_or_brown) %>%
-    tidyr::pivot_wider(
+    pivot_wider(
       names_from = .data$scenario,
       names_prefix = "weighted_production_target_",
-      values_from = .data$scenario_target_value
+      values_from = .data$scenario_target_value,
+      values_fn = list
```

Comment:
this is basically the only change, is that correct?

Reply:
The changes are two: 
  * `values_fn = list` suppresses the warning by which `pivot_wider()`  tells the user that it had created a list column, automatically.
  * `unnest()` (next line) then transforms the list column into a regular column.